Mathlib/Data/Rat/Cast/Lemmas.lean

```diff
@@ -20,6 +21,11 @@ namespace Rat
 
 variable {α : Type*} [DivisionRing α]
 
+@[simp, norm_cast]
+lemma cast_pow (p : ℚ) (n : ℕ) : ↑(p ^ n) = (p ^ n : α) := by
```

Comment:
Do we have the `zpow` version somewhere else?

Reply:
Yes, `NNRat.cast_zpow` in `Data.Rat.Cast.CharZero`